src/NServiceBus.Transport.RabbitMQ.TransportTests/App_Packages/NSB.TransportTests.6.1.1/NServiceBusTransportTest.cs

```diff
@@ -86,6 +86,8 @@ public void TearDown()
             ReceiveInfrastructure = TransportInfrastructure.ConfigureReceiveInfrastructure();
             SendInfrastructure = TransportInfrastructure.ConfigureSendInfrastructure();
 
+            await TransportInfrastructure.Start();
```

Comment:
Are we going to have the same problem on `develop`, but there we can't change the source files?

Reply:
Yes, but I've opened https://github.com/Particular/NServiceBus/issues/5099 to fix this in core. We'll have to work around it differently in the mean time.